### ISSUE TITLE
Fix travis.yml

### ISSUE THREAD

**Issue by ZER-0-NE** [NITDgpOS/Medium_Grabber#40]

**Comment by ZER-0-NE** [comment]
Sure @Melvin1Atieno Go ahead

**Comment by Melvin1Atieno** [comment]
I have been trying to get running with the grabber locally, following the instructions in the readme. I skipped the part where I needed to add my password to a pass.txt file. I figured since there really wasn't a  file and the code would prompt me for a password there was no need.
I am using chrome on Linux.

Error
![screenshot from 2018-12-19 16-38-37](https://user-images.githubusercontent.com/28667852/50223592-bce6a080-03ac-11e9-856e-a8c79f34d8dd.png)
